.circleci/config.yml

```diff
@@ -1,179 +1,31 @@
-      # Therefore we build this full nightly first, and then throw it away.
-      - run:
-          command: |
-            make clean
-            export VERSION=nightly.$(date +%Y%m%d%H%M%S)
-            export VERSION="$(git describe --tags --always --dirty)-nightly.$(date +%Y%m%d%H%M%S)"
-            echo VERSION=$VERSION
-            git submodule update --init && git submodule update --remote
-            make -f nightly_build.mak clean
-            make -f nightly_build.mak -s --print-directory VERSION=$VERSION DdevVersion=$VERSION DBTag=$VERSION DBATag=$VERSION WebTag=$VERSION RouterTag=$VERSION  NGINX_LOCAL_UPSTREAM_FPM7_REPO_TAG=$VERSION NGINX_LOCAL_UPSTREAM_FPM7_REPO_TAG=$VERSION UPSTREAM_PHP_REPO_TAG=$VERSION
-          no_output_timeout: "20m"
-          name: Run full nightly build
-
-      - run:
-          command: bin/linux/ddev version
-          name:  nightly-build ddev version information
-
-      # Run the built-in ddev tests with the executables just built.
-      - run:
-          command: make -s test
-          name: ddev tests
-          no_output_timeout: "20m"
-
-      - run: make -s gometalinter
-
-      # Now build using the regular ddev-only technique - this results in a fully clean set of executables.
-      # Earlier process updated submodules so now we clean them up to continue.
-      - run:
-          command: |
-            git submodule update --init &&
-            make -s clean linux darwin windows
-          name: Build the ddev executables
-
-      # Run the built-in ddev tests with the clean binaries just built.
-      - run:
-          command: make -s test
-          name: ddev tests
-          no_output_timeout: "20m"
-
-      - run:
-          command: bin/linux/ddev version
-          name: ddev version information (clean binaries, not nightlies)
-
-      - run:
-          command: ./.circleci/generate_artifacts.sh $ARTIFACTS
-          name: tar/zip up artifacts and make hashes
-
-      - store_artifacts:
-          path: /artifacts
-          name: Artifact storage
-
-  # 'tag_build' is used to build a tag for release.
-  tag_build:
-    machine:
-      image: circleci/classic:201711-01
     working_directory: ~/go/src/github.com/drud/ddev
-    environment:
-      DRUD_DEBUG: "true"
-      GOPATH: /home/circleci/go
-      ARTIFACTS: /artifacts
+
     steps:
       - run: mkdir -p ~/go/{lib,pkg,src/github.com/drud/ddev}
-
       - checkout
 
-      - run:
-          command: ./.circleci/circle_vm_setup.sh
-          name: TAG BUILD Circle VM setup - tools, docker, golang
-
-      # Now build using the regular ddev-only technique - this results in a fully clean set of executables.
-      - run:
-          command: make -s clean linux darwin windows
-          name: Build the ddev executables
-
-      - run:
-          command: bin/linux/ddev version
-          name: ddev version information
+      - run: |
+          brew cask install docker && brew install docker docker-compose golang
```

Comment:
You shouldn't have to `brew install docker docker-compose` here - the docker client and docker-compose both ship with DfM.

Reply:
Agreed. However, the install technique I'm doing there doesn't seem to get that in there. Or maybe it was one of those things where I was just confused.